### DISCUSSION TITLE
Players pressed ragdoll button should be able to speak normally

### DISCUSSION THREAD

**Discussion by Paragonid** [FakeFishGames/Barotrauma#7079] [Ideas and suggestions]
Currently players pressed ragdoll button can not speak (use voice chat).
It doesn't make much sense why they wouldn't be, and as it's supported way of doing some things like letting others to manipulate own items or use a water bike, it encourages us to use discord to overcome.

**Comment by StruggleDuck** [comment]
From what I am aware this is caused by the fact that by pressing space, the game simply applies the "stunned" effect to your character, and it is removed when you stop pressing space. I hope that they make this change, as it makes communication much more difficult, especially with the addition of new traits that involve the use of ragdolling to do certain tasks, whether it be tackling with beat cop, moving through the water with some other trait, or writing books. Fingers crossed.
